modules/genomic_browser/php/genomic_file.class.inc

```diff
@@ -78,18 +81,4 @@ class Genomic_File
             return null;
         }
     }
-
-    /**
-    * Gets the AnalysisProtocol for this file
-    *
-    * @return String from the file properties
-    */
-    function getAnalysisProtocol()
```

Comment:
This variable can be retrieved using `getParameter` like all the others. No need for a special function.